pkg/apis/componentconfig/cloudcore/v1alpha1/default.go

```diff
@@ -124,7 +126,17 @@ func NewDefaultCloudCoreConfig() *CloudCoreConfig {
 				Enable: true,
 			},
 		},
+		LeaderElection:	&componentbaseconfig.LeaderElectionConfiguration{
+			LeaderElect:		true,
```

Comment:
For the first version of ha, we ca set the `LeaderElect` to false. Which means ha is disable default.

Reply:
get it